### PR TITLE
[16.0][mig] account_move_line_product: Migration to 16.0

### PR DESCRIPTION
**Pull request by dsolanki-initos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ addon | version | maintainers | summary
 [stock_search_supplierinfo_code](stock_search_supplierinfo_code/) | 16.0.1.0.1 |  | Allows to search for picking from supplierinfo code
 [stock_storage_category_capacity_name](stock_storage_category_capacity_name/) | 16.0.1.0.0 |  | Allows to have a better display name for Stock Storage Category Capacity model
 [stock_warehouse_calendar](stock_warehouse_calendar/) | 16.0.1.0.0 | [![JordiBForgeFlow](https://github.com/JordiBForgeFlow.png?size=30px)](https://github.com/JordiBForgeFlow) | Adds a calendar to the Warehouse
+[account_move_line_product](account_move_line_product/) | 16.0.1.0.0 |  | Displays the product in the journal entries and items
```
Comment (rousseldenis) [inline README.md]: You can remove this. It will be updated automatically at merge

File: account_move_line_product/__manifest__.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Account Move Line Product",
-    "version": "15.0.1.0.1",
+    "version": "16.0.1.0.0",
```
Comment (rousseldenis) [inline account_move_line_product/__manifest__.py:6]: This change has to be set in migration commit instead pre-commit one.

**Comment by rousseldenis** [comment]
@dsolanki-initos Minor changes

File: README.md (hunk 1 above)
Comment (dsolanki-initos) [reply README.md]: Sure, I'll remove it and push it again.

File: account_move_line_product/__manifest__.py (hunk 2 above)
Comment (dsolanki-initos) [reply account_move_line_product/__manifest__.py:6]: Hi @rousseldenis, I am referring to OCA's standard steps. In those steps, I executed the pre-commit test locally, which indicated that the version should be 16.0.

Comment (rousseldenis) [reply account_move_line_product/__manifest__.py:6]: Yes but it says that you should commit with --no-verify :smiley: 

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0#how-to

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by CasVissers-360ERP — APPROVED** [review]
Functional review

**Pull request merged**